internal/postprocessor/go.mod

```diff
@@ -1,3 +1,46 @@
 module cloud.google.com/go/internal/postprocessor
 
 go 1.19
+
+replace cloud.google.com/go/internal/aliasfix => ../aliasfix
```

Comment:
nit: would you move this to the bottom of the file.

Reply:
Done